spec/mongo/client_spec.rb

```diff
@@ -1474,7 +1474,7 @@
       end
 
       let(:command) do
-        EventSubscriber.started_events.find { |c| c.command_name == :listDatabases }.command
+        EventSubscriber.started_events.find { |c| c.command_name == 'listDatabases' }.command
```

Comment:
I'm wondering if it might be worth adding an `EventSubscriber#command_started?` method (which could convert symbols to strings internally) given that we seem to be doing this operation a lot. Not worth changing for this PR, but something to think about in the future.

Reply:
A filter by command name would make sense, but per Bernie's comments we can't be adding any public methods to events in one driver.

Reply:
Can certainly do that at some point then.